ipn/ipnlocal/cert.go

```diff
@@ -458,14 +468,26 @@ func acmeKey(dir string) (crypto.Signer, error) {
 	return privKey, nil
 }
 
-func validCertPEM(domain string, keyPEM, certPEM []byte, now time.Time) bool {
+func validCertPEM(domain string, keyPEM, certPEM, extraRoots []byte, now time.Time) bool {
 	if len(keyPEM) == 0 || len(certPEM) == 0 {
 		return false
 	}
 	tlsCert, err := tls.X509KeyPair(certPEM, keyPEM)
 	if err != nil {
 		return false
 	}
+	roots, err := x509.SystemCertPool()
+	if err != nil {
+		log.Printf("cert check: failed to get system roots: %v", err)
+		return false
+	}
```

Comment:
Why is this a failure for a new path that should only be able tests?

Reply:
Oh, this was implicit before by being nil?

Reply:
Yes, the default if the option is nil is to use the system cert pool.